_ont/ont-technicolor-afm0003.md

```diff
@@ -8,27 +8,37 @@ alias: Hisense LTE3415-SH+
 
 # Hardware Specifications
 
-|             |                                                 |
-| ----------- | ----------------------------------------------- |
-| Vendor      | Technicolor                                     |
-| Model       | AFM0003TIM                                      |
-| Chipset     | Realtek RTL9601C                                |
-| Flash       | 256 MB                                          |
-| RAM         |                                                 |
-| System      | Linux 2.6 (Luna SDK 1.9)                        |
-| HSGMII      | Yes, but not working with stock firmware        |
-| Optics      |                                                 |
-| IP address  | 192.168.2.1                                     |
-| Web Gui     | Can be enabled, user `admin`, password `system` |
-| Telnet      | ✅                                              |
-| Form Factor | miniONT SFP                                     |
-| Multicast   | ✅                                              |
+|                  |                                                 |
+| ---------------- | ----------------------------------------------- |
+| Vendor/Brand     | Technicolor                                     |
+| Model            | AFM0003TIM                                      |
+| ODM              | Hisense                                         |
+| ODM Product Code | LTE3415-SCA+                                    |
```

Comment:
Are we sure? It uses a different chip and a different product name in technicolor side